contents/tutorials/cookieless-tracking.md

```diff
@@ -0,0 +1,92 @@
+---
+title: How to track users without using cookies
```

Comment:
```suggestion
title: How to use PostHog without cookies
```

Reply:
I feel like the current headline implies circumvention rather than the removal of cookie banners